### PR TITLE
:smug:

### PR DESCRIPTION
**Pull request by Manly-Dwarf**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl: Manly Dwarf (actually goonstation coders but sshhh)
add: You can now smug it up in chat with *smug. Complete with an emote. Spam it and you'll bite your tongue off.
/:cl:

## About The Pull Request
Shamelessly steals Goonstation code from 2014, after I was told that it was hardly used any more.
smug

## Why It's Good For The Game
:smug:

😏

### CODE REVIEW

File: hippiestation.dme [hunk 1]
```diff
@@ -3195,8 +3195,8 @@
 #include "hippiestation\code\modules\clothing\shoe.dm"
 #include "hippiestation\code\modules\clothing\suit.dm"
 #include "hippiestation\code\modules\clothing\under.dm"
-#include "hippiestation\code\modules\clothing\gloves\gloves.dm"
 #include "hippiestation\code\modules\clothing\glasses\glasses.dm"
+#include "hippiestation\code\modules\clothing\gloves\gloves.dm"
```
Comment (Manly-Dwarf) [inline hippiestation.dme:3199]: lol

File: hippiestation/code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -4,6 +4,8 @@
 	var/super_fart = 76
 	var/super_nova_fart = 12
 	var/fart_fly = 12
+	var/smug = 0
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/mob/living/carbon/human/emote.dm]: this var was supposed to be removed but I guess it's still here

**Comment by ExpAntique** [comment]
+1, :smug: :exhale:

**Comment by Majick1** [comment]
+1

**Comment by yobob591** [comment]
😏  +1

**Review by AnthonyJNgo — APPROVED** [review]
lgtm

**Comment by AnthonyJNgo** [comment]
😏

**Comment by lunaman22** [comment]
+1 *smug

**Pull request merged**